packages/vscode-formatter/src/components/title.ts

```diff
@@ -1,17 +1,26 @@
 import { compressToEncodedURIComponent } from "lz-string";
-import { Diagnostic } from "vscode-languageserver-types";
+import { Diagnostic, type Range } from "vscode-languageserver-types";
+import { URI } from "vscode-uri";
 import { d } from "@pretty-ts-errors/utils";
 import { KNOWN_ERROR_NUMBERS } from "./consts/knownErrorNumbers";
 import { miniLine } from "./miniLine";
 
-export const title = (diagnostic: Diagnostic) => d/*html*/ `
+export const title = (diagnostic: Diagnostic, uri?: URI) => d/*html*/ `
     <span style="color:#f96363;">⚠ Error </span>${
       typeof diagnostic.code === "number"
         ? d/*html*/ `
             <span style="color:#5f5f5f;">
-            (TS${diagnostic.code}) 
-            ${errorCodeExplanationLink(diagnostic.code)}  | 
+            (TS${diagnostic.code})
+            ${errorCodeExplanationLink(diagnostic.code)}  |
             ${errorMessageTranslationLink(diagnostic.message)}
+            ${
+              uri
+                ? `  | ${errorMessageOpenMarkdownPreview(
```

Comment:
We should show it only we're not already inside a preview panel right?

Reply:
Yes, I'm not sure how we can do that yet, as we are passing around the markdown contents as a string without any context of where it will be rendered.